tests/test_schemas.py

```diff
@@ -0,0 +1,303 @@
+import pytest
```

Comment:
For a new file, should we add the license declaration at the beginning like others?

Reply:
updated